tools/ete/ete_lineage_generator.xml

```diff
@@ -0,0 +1,143 @@
+$compresscond.lower
+#end if
+    ]]>
+    </command>
+<!--
+-->
+    <inputs>
+		<param name="speciesFile" type="data" format="txt" label="Species file" help="List with one species/taxid per line" />
+		<param name="database" type="data" format="sqlite" label="(ETE3) Taxonomy Database" help="The sqlite formatted Taxonomy used by ETE3 (which is derived from NCBI taxonomy)" />
+        <conditional name="ranks">
+            <param name="levels" type="select" label="Taxonomic levels">
+                <option value="primary" selected="true">Primary</option>
+                <option value="full">Full</option>
+                <option value="manual">Manual</option>
+            </param>
+            <when value="manual">
+			    <param name="manualranks" type="select" multiple="True" label="Manual selection of ranks">
+				    <option value="superkingdom">superkingdom</option>
+					<option value="kingdom" selected="True">kingdom</option>
+					<option value="subkingdom">subkingdom</option>
+                    <option value="superphylum">superphylum</option>
+					<option value="phylum" selected="True">phylum</option>
+					<option value="subphylum">subphylum</option>
+                    <option value="superclass">superclass</option>
+					<option value="class" selected="True">class</option>
+					<option value="subclass">subclass</option>
+						<option value="infraclass">infraclass</option>
+					<option value="cohort">cohort</option>
+                    <option value="superorder">superorder</option>
+			        <option value="order" selected="True">order</option>
+					<option value="suborder">suborder</option>
+					<option value="infraorder">infraorder</option>
+					<option value="parvorder">parvorder</option>
+                    <option value="superfamily">superfamily</option>
+			        <option value="family" selected="True">family</option>
+					<option value="subfamily">subfamily</option>
+	                    <option value="tribe">tribe</option>
+			        <option value="subtribe">subtribe</option>
+                    <option value="genus" selected="True">genus</option>
+			        <option value="subgenus">subgenus</option>
+                    <option value="species group">species group</option>
+			        <option value="species subgroup">species subgroup</option>
+					<option value="species" selected="True">species</option>
+					<option value="subspecies">subspecies</option>
+                    <option value="varietas">varietas</option>
+			        <option value="forma">forma</option>
+	            </param>
+            </when>
+	</conditional>
+		<conditional name="compresscond">
+				<param name="compress" type="boolean" label="Fill unnamed ranks" truevalue="-c" falsevalue="" help="Fill unnamed ranks with super/sub ranks (see -l)" />
+				<when value="-c">
+					<param name="lower" type="boolean" label="Prefer lower ranks for filling" truevalue="-l" falsevalue="" help="Prefer lower levels for filling" />
```

Comment:
`label` and `help` should always be the second-to-last and last attributes respectively, according to IUC [coding style](https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html#coding-style). Rationale is that if the XML element is long, `label` and `help` attributes can be on a new line(s).